src/applications/vaos/tests/new-appointment/components/DateTimeSelectPage/index.unit.spec.jsx

```diff
@@ -671,7 +673,18 @@ describe('VAOS <DateTimeSelectPage>', () => {
       clinics,
       pastClinics: true,
     });
-    const slot308Date = moment();
+
+    await setTypeOfCare(store, /primary care/i);
```

Comment:
I rearranged some of the code in here to read better with the Given/When/Then comments. The only functional change is the addition of the extra slot